### ISSUE TITLE
After upgrade to version 3.0.0 (from 2.7.0) settings.polyfills is not respected anymore

### ISSUE THREAD

**Issue by pkuczynski** [amilajack/eslint-plugin-compat#187]
I have in my eslint following setting:

```
  settings: {
      polyfills: [
      'fetch',  // IE11
      'object-values', // IE11
      'promises' // IE11
    ],
    react: {
      version: packageJson.dependencies.react.replace('^', '')
    }
  },
```

When using version 2.7.0 no errors reported. After upgrade to 3.0.0:

```
xxx/yyy/zzz.js
  13:11  error  Promise.all() is not supported in IE 11  compat/compat

aaa/bbb/cc.js
  7:16  error  Object.values() is not supported in IE 11  compat/compat
```

**Comment by amilajack** [comment]
Try changing your `settings` config to:
```
{
 settings: {
      polyfills: [
      'fetch',  // IE11
      'Object.values', // IE11
      'promises' // IE11
    ]
  },
```

**Comment by pkuczynski** [comment]
How would that fix the issue with `Promise.all()`? And why it was working in the previous versions?

**Comment by amilajack** [comment]
Oh my bad. Just updated my commend and fixed the config

**Comment by niftylettuce** [comment]
Why was this removed?  There doesn't seem to be a workaround yet for this unless I'm wrong?

Also the docs need updated in README, it's saying you CAN do this in v3 but you can't.

![Screen Shot 2019-03-18 at 3 43 14 PM](https://user-images.githubusercontent.com/523141/54562283-b3ada580-4994-11e9-9b29-d3644332b8f6.png)

**Comment by amilajack** [comment]
> Also the docs need updated in README, it's saying you CAN do this in v3 but you can't.

Can you please elaborate on this

**Comment by pkuczynski** [comment]
Confirmed it works!
